Add additional binary types

jlloyd-widen opened the pull request:
This addresses #66 

- [ x] New feature (non-breaking change which adds functionality)

## Checklist

- [ x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [ ] CI checks pass with my changes
- [ x] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [ ] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [ ] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ x] Relevant documentation is updated including usage instructions

Samira-El commented:
Hi @jlloyd-widen , thanks for the contribution!

Can you please add unit and integration tests that cover these new types? thx

jlloyd-widen commented:
> Can you please add unit and integration tests that cover these new types? thx

@samira-el I tried running unit and integrations test (didn't even try adding new ones) and I couldn't get them to pass successfully. The instructions in the readme did not work for me. `nosetest` didn't actually run any tests and `pytest` fails half the tests right from the get-go. Until I have a way to pass all existing tests, I don't see a point to adding new ones. Some guidance on how to run these tests successfully would be appreciated.

Samira-El commented:
Hey, not sure what you meant by instructions didn't work for you and nosetests didn't run any tests.

Please try running again with the same commands as the [CI job](https://github.com/transferwise/pipelinewise-tap-mysql/blob/master/.github/workflows/ci.yaml).

jlloyd-widen commented:
@Samira-El I've updated my code to incorporate your most recent changes and I've added a few integration tests regarding the blob, and binary types. I also fixed a few of the other data type integration tests that appeared to be failing. let me know what you think.
